README.md

```diff
@@ -533,123 +533,153 @@ The Embedded SDK for MAX78000 and MAX78002 is used to compile, flash, and debug
-
-OpenOCD binaries are available in the “openocd” sub-folder of the ai8x-synthesis repository. However, some additional dependencies are required on most systems. See [openocd/Readme.md](openocd/Readme.md) for a list of packages to install, then return here to continue.
-
-##### System Path
+Once the tools above have been installed, continue with [Final Check](#Final-Check).
 
-Next, add the location of the toolchain binaries to the system path.
+#### Method 2: Manual Installation
 
-For example, on Linux add the following to `~/.profile` (and on macOS, to `~/.zprofile`), adjusting for the actual `PATH` to the compilers and the system’s architecture (`TARGET_ARCH`):
+The MAX78000/MAX78002 SDK is available as a git submodule of ai8x-synthesis. It is checked out automatically to a version compatible with the project into the folder `ai8x-synthesis/sdk`. This submodule contains all of the SDK's components _except_ the Arm GCC, RISC-V GCC, and Make. These must be downloaded and installed manually.
 
-```shell
-# Arm GCC
-ARMGCC_DIR=/usr/local/gcc-arm-none-eabi-10.3-2021.10  # Change me!
-echo $PATH | grep -q -s "$ARMGCC_DIR/bin"
-if [ $? -eq 1 ] ; then
-    PATH=$PATH:"$ARMGCC_DIR/bin"
-    export PATH
-    export ARMGCC_DIR
-fi
-
-# RISC-V GCC
-RISCVGCC_DIR=/usr/local/xpack-riscv-none-embed-gcc-10.2.0-1.2  # Change me!
-echo $PATH | grep -q -s "$RISCVGCC_DIR/bin"
-if [ $? -eq 1 ] ; then
-    PATH=$PATH:"$RISCVGCC_DIR/bin"
-    export PATH
-    export RISCVGCC_DIR
-fi
-
-# OpenOCD
-OPENOCD_DIR=$HOME/Documents/Source/ai8x-synthesis/openocd/bin/TARGET_ARCH  # Change me!
-echo $PATH | grep -q -s "$OPENOCD_DIR"
-if [ $? -eq 1 ] ; then
-    PATH=$PATH:$OPENOCD_DIR
-    export PATH
-    export OPENOCD_DIR
-fi
-```
+1. Download and install the Arm Embedded GNU Toolchain from [https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads).
+
+    * Recommended version: 10.3-2021.10 *(newer versions may or may not work correctly)*
+    * Recommended installation location: `/usr/local/gcc-arm-none-eabi-10.3-2021.10/`
+
+2. Download and install the RISC-V Embedded GNU Toolchain from [https://github.com/xpack-dev-tools/riscv-none-embed-gcc-xpack/releases/](https://github.com/xpack-dev-tools/riscv-none-embed-gcc-xpack/releases/)
+
+    * Recommended version: 10.2.0-1.2 *(newer versions may or may not work correctly)*
+    * Recommended installation location: `/usr/local/riscv-none-embed-gcc/10.2.0-1.2/`
+
+3. Install GNU Make
+    * (Linux/macOS) “make” is available on most systems by default. If not, it can be installed via the system package manager.
+    * (Windows) Install [MSYS2](https://www.msys2.org/) first, then install “make” using the MSYS2 package manager:
+
+      ```shell
+      $ pacman -S --needed make
```

Comment:
Change to `$ pacman -S --needed base filesystem make msys2-runtime` to match packages with the MaximSDK